buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -599,8 +599,12 @@ class BeamModulePlugin implements Plugin<Project> {
         testApt auto_service
 
         // These dependencies are needed to avoid error-prone warnings on package-info.java files,
-        // also to include the annotations to supress warnings.
-        def findbugs_annotations = "com.github.stephenc.findbugs:findbugs-annotations:1.3.9-1"
+        // also to include the annotations to suppress warnings.
+        //
+        // findbugs-annotations artifact is licensed under LGPL and cannot be included in the
+        // Apache Beam distribution, but may be relied on during build.
+        // See: https://www.apache.org/legal/resolved.html#prohibited
+        def findbugs_annotations = "com.google.code.findbugs:annotations:3.0.1"
```

Comment:
Findbugs annotations are retained in the class file, as it operates on bytecode not source code. It may fall under https://www.apache.org/legal/resolved.html#optional.

Reply:
In this case we aren't using the annotations in our own Java source code; [our byte-buddy dependency](https://github.com/apache/beam/blob/d75365eeeff8d8b3fc6c97e582854ddcc9da49b6/sdks/java/core/build.gradle#L23) uses them but [declares `findbugs-annotations` as a `provided` dependency](https://github.com/raphw/byte-buddy/blob/471739c535aa2c17faf5fb5eafc4231e418e3796/pom.xml#L124), so we need to also declare it to resolve their usages..

I think this is ok as long as we don't accidentally introduce a usage down-the-road. Do you think we need an [API surface test](https://github.com/apache/beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/util/ApiSurfaceTest.java)?

Reply:
Note: I attempted writing an API surface test, which failed because the `@SuppressFBWarning` annotation has [`RetentionPolicy.CLASS`](https://docs.oracle.com/javase/7/docs/api/java/lang/annotation/RetentionPolicy.html#CLASS) and is erased during test execution.

Perhaps I can create a checkstyle rule instead..

Reply:
Adding a checkstyle rule worked: [sample build scan](https://gradle.com/s/vhzbhydcifvkm).

Reply:
Hmm, interesting. I'm still not clear that we have to ban it, but since it is green that's great.